### ISSUE TITLE
matplotlib 3d interactive plot boundaries

### ISSUE THREAD

**Issue by SaHaddad** [matplotlib/matplotlib#24679]
Hi I want to zoom in more on the plot but the curve is cutted out by a "frame" as shown in the picture below and i would like to see the curve fill all the windows i asked help on python discord but couldnt get an answer any help would be greatly appreciated
![image](https://user-images.githubusercontent.com/100862816/206780241-56ea9473-e179-4a3c-88d1-d36872948e84.png)

**Comment by oscargus** [comment]
The box aspect is fixed in 3D plots, but you can change it by calling `set_box_aspect`. As opposed to the 2D case, setting it to None will not make it adapt, but that seems like a reasonable request, although it will require a bit of ad-hoc since there is not a unique way to figure out how to fill the window.

**Comment by SaHaddad** [comment]
@oscargus 
Sorry but what should i put in the setboxaspect thing exactly ?
Here is my code
```python import matplotlib.pyplot as pl
from mpl_toolkits.mplot3d import Axes3D
import math as mt
def attracteur(x,y,z,dt):
    lx,ly,lz,=[x],[y],[z]
    t=0
    β,ρ,σ=8/3,28,10
    while t<100:
        x,y,z=(x+dt*(σ*(y-x))),(y+dt*(ρ*x-y-x*z)),(z+dt*(y*x-β*z))
        t+=dt
        lx.append(x)
        ly.append(y)
        lz.append(z)
    pl.figure()
    ax=pl.axes(projection='3d')
    ax.plot3D(lx,ly,lz,'b',linewidth=0.5)
    pl.title('')
    ax.set_ylabel('$y$')
    ax.set_xlabel('$x$')
    ax.set_zlabel('$z$')
    pl.show()
```
and a video will explain itself
[2022-12-09_19-43-38.webm](https://user-images.githubusercontent.com/100862816/209472679-be87f265-640a-414b-b92d-87f381a23feb.webm)

**Comment by oscargus** [comment]
Sorry, it didn't work as I expected. One can affect it, but I cannot really find a way to expand it to cover the whole frame.

**Comment by QuLogic** [comment]
I think this is fixed by #27349.

**Comment by scottshambaugh** [comment]
I don't believe this is related to https://github.com/matplotlib/matplotlib/pull/27349 - my interpretation of the original issue is that the user wants to expand the *2D* clip box to fill the entire figure, like so
<img width="602" alt="image" src="https://github.com/user-attachments/assets/5ea0b48d-70ec-48a0-8950-38f8b60d62f8">

**Comment by SaHaddad** [comment]
Wow ! To be honest i was surprised to get any emails out of that issue recently. And yes what the previous comment illustrates is exactly what i was asking about. Are they still no commands to do it ?

**Comment by SaHaddad** [comment]
No worries this is just a nit pick.
